app/views/posts/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Here is a particular post by a specific user</h1>
\ No newline at end of file
+<section class="all-users">
+  <div class="post_desc">
+    <div class="unit">
+      <h2><%= @post.title %> by <%= @post.author.name%></h2>
+      <h4>Comments:<%= @post.comments_counter%>,  Likes:<%= @post.likes_counter%></h4>
+    </div>
+    <p class="P_text"><%= @post.text%></p>
+  </div>
+  
+  <div class="com_on_Post">
+  <h3>comments:</h3>
+    <ul>
+      <% @post_comments.each do |comment| %>
+        <li><%= comment.author.name %>: <%= comment.text %></li>
+      <% end %>
+    </ul>
+  </div>
+</section>
```

Comment:
- [ ] Kindly make sure that the post show page matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png) and move comments and likes to the right as given 🤓📌

![image](https://github.com/jonas-45/blog-app/assets/106119105/a72079d2-16dd-44d0-8d35-4131f1e3aef4)